bedrock/firefox/templates/firefox/features/pdf-free-fr.html

```diff
@@ -83,6 +83,6 @@ <h2>Adoptez la simplicité et la sécurité avec l'éditeur PDF Firefox</h2>
 
 <p>Découvrez comment Firefox peut rendre vos tâches quotidiennes plus simples et plus sécurisées.</p>
 
-<p>Téléchargez Firefox dès aujourd'hui et reprenez le contrôle de vos documents PDF&nbsp;!</p>
+<p><a href="{{ url('firefox.new') }}" data-link-text="Download Firefox" data-link-position="content">Téléchargez Firefox</a> dès aujourd'hui et reprenez le contrôle de vos documents PDF&nbsp;!</p>
```

Comment:
**Nit:** We should also probably hide this line from Firefox browsers

Reply:
✅